src/proxy-ui-api/frontend/src/components/KeysTable.vue

```diff
@@ -17,7 +17,7 @@
           <div class="clickable-link" @click="keyClick(key)">{{key.name}}</div>
         </div>
         <tr v-for="cert in key.certificates" v-bind:key="cert.id">
-          <td>
+          <td class="td-name">
```

Comment:
In general level I'd say using elements name in the css-class is a bad practise. If you would ever need to change to html structure you would have to change the css-class name also. 

But changing implementation is not required here.